docs/governance/templates/release_issue.md

```diff
@@ -0,0 +1,54 @@
+# Release {version}
+
+<!--
+This is the release issue template. Make a copy of the markdown in this page
+and copy it into a release issue. Fill in relevent values, found inside {}
+!-->
+Complete Milestone
```

Comment:
This should indicate what needs to be done for release candidates, main releases, and hotfixes.